### DISCUSSION TITLE
Include practical examples in README instead of Bear example

### DISCUSSION THREAD

**Discussion by prmichaelsen** [pmndrs/zustand#2493] [Ideas] [answered]
It would be nice if the docs demonstrated the concepts with closer to real world examples, such as maybe a product order tracker or an add to cart feature or something. It's hard for me to follow examples that are totally made up because my brain is doing extra work to figure out what the application is trying to do while also learning how it's doing it. Real world examples don't require me to guess what the examples are trying to accomplish.

**Comment by dai-shi** [comment, marked as answer]
Thanks for the feedback. It's intentional to have the bear example in README, but we may consider adding some other examples in docs. cc @dbritto-dev

**Reply by dbritto-dev** [reply]
@prmichaelsen @dai-shi totally, actually I've been thinking adding a tutorial and getting started guides not just few code examples

**Reply by prmichaelsen** [reply]
Thank you, that would be great. Let me know if you need any contributors.

**Reply by dbritto-dev** [reply]
@prmichaelsen working on that I'll you know when that is finish but for now you can track the changes here: https://github.com/pmndrs/zustand/pull/2508

**Reply by dbritto-dev** [reply]
btw, for now I'm just structuring the content
